src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp

```diff
@@ -186,9 +186,9 @@ bool QJsonWebToken::setAlgorithmStr(QString strAlgorithm)
 	// set algorithm
 	m_strAlgorithm = strAlgorithm;
 	// modify header
-	m_jdocHeader = QJsonDocument::fromJson(QObject::trUtf8("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
+	m_jdocHeader = QJsonDocument::fromJson(QObject::tr("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
 		                                 + m_strAlgorithm.toUtf8()
-		                                 + QObject::trUtf8("\"}").toUtf8());
+		                                 + QObject::tr("\"}").toUtf8());
```

Comment:
Why this is being translated ?

Reply:
I raised the same question in my commit message ;) I think it's totally unnecessary.

I'm a bit tired fighting with qgc and all the changes that brake my local build when I do a rebase every other day. Feel free to adopt the pull request.